components/Shop/ShopTable.js

```diff
@@ -18,29 +19,73 @@ const StyledImageContainer = styled.div`
   gap: 5px;
 `;
 
-export default function ShopTable({ data, onItemClick }) {
+const StyledShopTableWrapper = styled.section`
```

Comment:
There are a few semantic considerations to keep in mind:
[`<section>`](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/section?retiredLocale=de)elements are expected to have a heading (`<h1>-<h6>`) that describes the theme of the section, so if there's no such heading, `<div>` could actually be a better choice.